js/functions.js

```diff
@@ -0,0 +1,68 @@
+/*
+Иван, добрый день! Подскажите, такой вариант первой функции будет верный? :
+const getLengthCheck = (string, maxLength) => { (string.length <= maxLength) ? true : false};
+Проверял в консоли Chrome и всегда возвращалось "undefined".
+Работало только так: const getLengthCheck = (string, maxLength) => { (string.length <= maxLength) ? console.log(true) : console.log(false)};
+*/
+//Функция для проверки длины строки:
+const getLengthCheck = (string, maxLength) => {
+  if (string.length <= maxLength) {
+    return true;
+  }
+  return false;
+}
+
+//Функця для проверки, является ли строка палинтромом + случай, когда в строке встречаются пробелы:
+const getpalindromCheck = (string) => {
+  let newString = string.toLowerCase();
+    newString = newString.replaceAll(' ', '');
+  let mirrorString = '';
+    for (i = newString.length - 1; i >= 0; i--) {
```

Comment:
Любую переменную перед использованием обязательно нужно объявлять через let/const, в данном случае она (переменная `i`) у нас меняется (мы делаем декремент её значения `i--`) поэтому нужен 'let' 
```suggestion
  for (let i = newString.length - 1; i >= 0; i--) {
```

Так нельзя делать, потому что без явного указания переменная работать будет, но она будет неявно объявлена через `var` это уже устаревший варивант, такие переменные объявляются глобально,  а не на уровне блока кода и могут перезаписаться в любой момент и есть много подводных камней Их вообще лучше никогда не использовать.